herbert_core/utilities/misc/make_const_bin_boundaries_descr.m

```diff
@@ -0,0 +1,25 @@
+function pbin = make_const_bin_boundaries_descr(p)
+% Make a constant bin boundaries descriptor from the bin boundaries
```

Comment:
Maybe worth extending this comment to make it clear the way this works with `make_const_bin_boundaries`

```
pbin_initial = [1, 1, 10];
p=make_const_bin_boundaries(pbin_initial );
p_desc = make_const_bin_boundaries_descr(p);

% p_desc is equal to pbin_initial (to within rounding errors)
```